### DISCUSSION TITLE
triggering an action biweekly (once every two weeks)

### DISCUSSION THREAD

**Discussion by roddi** [community/community#49483] [Actions]
### Select Topic Area

Bug

### Body

Hi,

seem like you cannot trigger a workflow **exactly** every two weeks. Any insights?

Best Roddi

**Comment by jge162** [comment]
Are you using cron? what are you using to run (once every two weeks). You have not provided enough detail to properly assist you. thanks.

**Reply by roddi** [reply]
I would use cron if that solved my problem but it doesn't.

**Reply by jge162** [reply]
This company allows you to schedule GitHub actions. If cron does not work for you. I have not personally used them (because crons works for me) but it is an option.

https://zapier.com/apps/schedule/integrations/github

![image](https://user-images.githubusercontent.com/31228460/223696462-f7510807-37ee-4455-90e2-6a989f0f3e1a.png)

**Comment by wham** [comment]
You can find a working solution at https://github.com/wham/bi-weekly-action. Hope it helps.
